core/src/apps/webauthn/__init__.py

```diff
@@ -18,4 +18,7 @@ def boot(iface: io.HID) -> None:
         __name__,
         "remove_resident_credential",
     )
-    loop.schedule(handle_reports(iface))
+    if not __debug__ or utils.EMULATOR:
+        import usb
```

Comment:
perhaps `import usb` can be on top level, seeing as `usb` is imported in all cases anyway

Reply:
Can be..